II.pass-tests/03-sum-numbers/add-numbers.js

```diff
@@ -1,4 +1,5 @@
 function addNumbers(numbers) {
-
+    var sum=0;
```

Comment:
This line is redundant. `sum` below is defined as a parameter to the callback and supplied for you. If you want to *override* the starting value of the accumulator, you can pass that as the second argument to `reduce` (e.g. `numbers.reduce((a, b) => a * b, 1)` would calculate the product, rather than the sum, of the numbers).